Find a functionally equivalent code:android.media.MediaCodecInfo.isEncoder:COMMENT
Method Modifier: public      final       
Comment:/**
 * Query if the codec is an encoder.
 */

@@ -1,3 +1,3 @@
 {
-    return MediaCodecList.isEncoder(mIndex);
+    return mIsEncoder;
 }

